### PR TITLE
chore:  add endpoints for local node data 

### PR DESCRIPTION
**Pull request by Sotatek-HuyLe3a**

### CODE REVIEW

File: stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalCommitteeController.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@RequestMapping("${apiPrefix}/local/committees")
```
Comment (satran004) [inline stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalCommitteeController.java:21]: Should we also add governance prefix?

${apiPrefix}/governance/local/committees

File: stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalCommitteeController.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+@RequiredArgsConstructor
+@Tag(name = "Local Committee Service", description = "Get committee info directly from local Cardano Node through n2c local query.")
```
Comment (satran004) [inline stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalCommitteeController.java:23]: "Get committee info directly from Cardano node through n2c local query"

Fyi. Remote Cardano n2c can be accessed through proxy like socat.

File: stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalDRepDistrController.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+@Tag(name = "Local Constitution Service", description = "Get constitution directly from local Cardano Node through n2c local query.")
+@Slf4j
+@ConditionalOnBean(LocalGovStateService.class)
+@ConditionalOnExpression("${store.epoch.endpoints.drep.local.enabled:true}")
+public class LocalDRepDistrController {
+    private final LocalDRepDistrService localDRepDistrService;
+
+    @GetMapping("/{dRepHash}/stake")
+    @Operation(description = "Get dRep hash distribution")
+    public ResponseEntity<LocalDRepStakeDto> getCommitteeInfo(@PathVariable String dRepHash, @RequestParam(name = "epoch") Integer epoch) {
```
Comment (satran004) [inline stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalDRepDistrController.java:29]: Typo in method name. It should be getDRepStakeDistr...

File: stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalDRepDistrController.java [hunk 4]
```diff
@@ -15,12 +15,12 @@
 import org.springframework.web.server.ResponseStatusException;
 
 @RestController
-@RequestMapping("${apiPrefix}/local/dRep")
+@RequestMapping("${apiPrefix}/governance/live/dRep")
```
Comment (satran004) [inline stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalDRepDistrController.java]: @Sotatek-HuyLe3a Can you please change dRep to lower case "drep"?

I think we should change it to plural "dreps" to align with other controllers like "blocks", "scripts" etc.

**Comment by satran004** [comment]
@Sotatek-HuyLe3a We need to add the the live endpoint config to governance starter.

```
store.epoch.endpoints.committee.live.enabled
```

File: stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalCommitteeController.java (hunk 1 above)
Comment (Sotatek-HuyLe3a) [reply stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalCommitteeController.java]: I fixed this

File: stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalCommitteeController.java (hunk 2 above)
Comment (Sotatek-HuyLe3a) [reply stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalCommitteeController.java]: I fixed this

File: stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalDRepDistrController.java (hunk 3 above)
Comment (Sotatek-HuyLe3a) [reply stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalDRepDistrController.java]: i fixed this

File: stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalDRepDistrController.java (hunk 4 above)
Comment (Sotatek-HuyLe3a) [reply stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalDRepDistrController.java]: i fixed this

**Comment by Sotatek-HuyLe3a** [comment]
> @Sotatek-HuyLe3a We need to add the the live endpoint config to governance starter.
> 
> ```
> store.epoch.endpoints.committee.live.enabled
> ```

I've added the live endpoint config

**Review by satran004 — APPROVED** [review]
Thanks @Sotatek-HuyLe3a. Looks good.

**Pull request merged**